apps/site/src/app/(home)/sections/Landing/Title.module.scss

```diff
@@ -1,4 +1,11 @@
+@use "bootstrap-utils" as bootstrap;
+
 .title {
-	width: clamp(0px, 40vw, 40vw);
-	height: clamp(0px, 14.53vw, 14.53vw);
+	width: clamp(0px, 70vw, 70vw);
```

Comment:
I think this can be resolved to just max(0px, 70vw)

Reply:
however both work so nothing's actually wrong

Reply:
Oh wait clamp isn't needed here. It was from an old artifact, so I will remove.